doc/GettingStarted.md.in

```diff
@@ -3,60 +3,60 @@ VE Offloading framework (VEO) is a framework to provide accelerator-style
 programming on Vector Engine (VE).
 
 ## Introduction
-Recently the accelerator programming model has become popular due to
-the wide usage of GPGPU and the increasing use of Xeon Phi processors.
+SX-Aurora TSUBASA provides VE Offloading framework (VEO) for
+the accelerator programming model.
 The accelerator programming model executes parallelized and/or vectorized
 numeric code such as matrix operations on accelerators and a main code
 controlling accelerators and performing I/O on a host.
-This programming model explicitly coded into parallel frameworks such as
-OpenMP 4 (device construct) and OpenACC, and into lower level APIs
-such as OpenCL and CUDA.
 
-SX-Aurora TSUBASA provides VE Offloading framework (VEO) for
-the accelerator programming model and enables to develop a program
-using both VEs and VH computing resources.
+The Alternative VE Offloading framework (AVEO) is a faster and much
+lower latency replacement to the previous VEO implementation which
+brings multi-VE support, simultaneous debugging of VE and VH side, API
+extensions.
+
+You can migrate to AVEO from the previous VEO implementation by
+installing the AVEO's packages and re-linking your program with AVEO
+without modification of makefiles.
 
-## Hello World
-First, let's try a "Hello, World!" program on VE.
 
-### Install required packages
-To run programs using VEO, please install aveo
-and runtime packages of the compiler.
+## Installation
 
-To develop programs using VEO, aveo-devel
-and development packages of compiler are also required.
+### Installing runtime package
 
-To install the packages to run VEO programs by yum, execute
-the following command as root:
+To run programs, please install @PACKAGE@ and @PACKAGE@run 
+and the runtime packages of the compiler (2.2.2 or later).
+
+To install the packages to run programs by yum, execute the following
+command as root:
 
 ~~~
-# yum install aveo
+# yum install @PACKAGE@ @PACKAGE@run
 ~~~
+You need not uninstall veoffload and veoffload-veorun which are the
+packages of the previous VEO implementation. So, you can execute
+your program linked with the previous VEO implementation.
+
+
+### Installing development package
 
-veoffload-veorun is automatically installed by dependency.
+To develop programs, @PACKAGE@-devel and @PACKAGE@run-devel and
+thedevelopment packages of the compiler (2.2.2 or later) are also
```

Comment:
type, space missing

Reply:
Typo was already fixed in commit :4497f91.

https://github.com/veos-sxarr-NEC/aveo/commit/4497f91f4063eacb2b4bc22382490f3d64caf0de